base_img/etc/lighttpd/lighttpd.conf

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
I'm not as familiar with httpd configs myself, but it might be good to run this by other folks at Delphix who are if you haven't done so already. The security team / whoever worked on nginx might be good candidates.

Reply:
I have asked Raul to take a look at this.